src/joint.jl

```diff
@@ -1,3 +1,22 @@
+struct Bounds{T}
+    lower::T
+    upper::T
+end
+
+struct JointBounds{T}
+    position::Bounds{Vector{T}}
+    velocity::Bounds{Vector{T}}
+    effort::Bounds{Vector{T}}
+end
+
+function JointBounds(::JT) where {T, JT <: JointType{T}}
+    p = Bounds(fill(typemin(T), num_positions(JT)), fill(typemax(T), num_positions(JT)))
```

Comment:
Add convenience constructor: `Bounds(min::T, max::T, length::Int) = Bounds(fill(min, length), fill(max, length))`.

Reply:
(I assume this is no longer relevant now that Bounds is scalar-like)